### PR TITLE
Public Release for Fall24 meta-release

### PR DESCRIPTION
**Pull request by bigludo7**
According to the discussion in: [V0.2.0 M4 by FabrizioMoggio · Pull Request #95 · camaraproject/CallForwardingSignal (github.com)](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fcamaraproject%2FCallForwardingSignal%2Fpull%2F95&data=05%7C02%7Cludovic.robert%40orange.com%7C4c5691990c9c4d1ba7f108dccced11a1%7C90c7a20af34b40bfbc48b9253b6f5d20%7C0%7C0%7C638610565788426084%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=KqTlZKf99AKrZmyiu3ZDWn10Iq27QtKFGhxTkUik74s%3D&reserved=0)

M4 version for Fall24:

- OAS file renamed according to: https://github.com/camaraproject/CallForwardingSignal/issues/93
- Remove “-rc.n” from the version and server path: https://github.com/camaraproject/CallForwardingSignal/discussions/92
- Remove "-rc.02" from "x-camara-commonalities": https://github.com/camaraproject/CallForwardingSignal/issues/94
- updated changelog.md
- updated readme.md
- updated .feature files according to: https://github.com/camaraproject/CallForwardingSignal/issues/96
- updated Readiness CL according to: https://github.com/camaraproject/CallForwardingSignal/issues/97
- updated changelog.md according to: https://github.com/camaraproject/CallForwardingSignal/issues/98
- updated readme.md according to: https://github.com/camaraproject/CallForwardingSignal/issues/100

#### What type of PR is this?

Add one of the following kinds:

* subproject management
* cleanup
* documentation

#### What this PR does / why we need it:

final release for Fall24

#### Which issue(s) this PR fixes:

https://github.com/camaraproject/CallForwardingSignal/issues/94
https://github.com/camaraproject/CallForwardingSignal/issues/93
https://github.com/camaraproject/CallForwardingSignal/issues/96
https://github.com/camaraproject/CallForwardingSignal/issues/97
https://github.com/camaraproject/CallForwardingSignal/issues/98
https://github.com/camaraproject/CallForwardingSignal/issues/100

#### Special notes for reviewers:

PR done on behalf of @FabrizioMoggio

#### Changelog input

```
 release-note

```

#### Additional documentation 

This section can be blank.

```
docs

```

### CODE REVIEW

**Comment by bigludo7** [comment]
@tanjadegroot For context: @FabrizioMoggio has issue with easyCLA so I copied his PR.
I fixed your review comments.
Regarding your 2 questions from the yaml:

on line 229: does unconditional forwarding overwrite conditional forwarding if both are active ?
Ludo: I tend to think that unconditional has predominance and override all other but happy to listen from @FabrizioMoggio 

on line 230: with "This endpoint exceeds the main scope of the CFS API, for this reason an error code 501 can be returned" do you mean that this second endpoint is optional and may not be provided and that 501 is returned in that case ?
Ludo: Yes

File: code/API_definitions/Call_Forwarding_Signal.yaml
Comment (hdamker) [inline code/API_definitions/Call_Forwarding_Signal.yaml:1]: @bigludo7 the filename need to be changed to call-forwarding-signal.yaml (as done in #95)

File: CHANGELOG.md [hunk 1]
```diff
@@ -60,9 +91,9 @@ The pre-release is suitable for implementers, but it is not recommended to use t
 The version number for the YAML file is v0.2.0-rc.1
 
 - API definition **with inline documentation**:
-  - OpenAPI [YAML spec file](https://github.com/camaraproject/CallForwardingSignal/blob/r1.1/code/API_definitions/Call_Forwarding_Signal.yaml)
-  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/CallForwardingSignal/r1.1/code/API_definitions/Call_Forwarding_Signal.yaml&nocors)
-  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/CallForwardingSignal/r1.1/code/API_definitions/Call_Forwarding_Signal.yaml)
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/CallForwardingSignal/blob/r1.1/code/API_definitions/call-forwarding-signal.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/CallForwardingSignal/r1.1/code/API_definitions/call-forwarding-signal.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/CallForwardingSignal/r1.1/code/API_definitions/call-forwarding-signal.yaml)
```
Comment (hdamker) [inline CHANGELOG.md:96]: @bigludo7 here in r1.1 the filename will stay unchanged ... the links work and will work perfectly. Don't try to change the history 😄

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +15,36 @@ The below sections record the changes for each API version in each release as fo
...
+* Identity and Consent Management v0.2.0
+
+## call-forwarding-signal v0.2.0
+
+This is the first official release of the Call Forwarding Signal API, version 0.2.0. It contains mainly alignments with the Commonalities 0.4.0 and the Release Management Guidelines. 
+
+- API definition **with inline documentation**:
+  - OpenAPI [YAML spec file](https://github.com/camaraproject/CallForwardingSignal/blob/r1.3/code/API_definitions/Call_Forwarding_Signal.yaml)
+  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/CallForwardingSignal/r1.3/code/API_definitions/Call_Forwarding_Signal.yaml&nocors)
+  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/CallForwardingSignal/r1.3/code/API_definitions/Call_Forwarding_Signal.yaml)
```
Comment (hdamker) [inline CHANGELOG.md:36]: Here you need to change filename in the links:
```suggestion
  - OpenAPI [YAML spec file](https://github.com/camaraproject/CallForwardingSignal/blob/r1.3/code/API_definitions/call-forwarding-signal.yaml)
  - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/CallForwardingSignal/r1.3/code/API_definitions/call-forwarding-signal.yaml&nocors)
  - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/CallForwardingSignal/r1.3/code/API_definitions/call-forwarding-signal.yaml)
```

File: code/API_definitions/Call_Forwarding_Signal.yaml
Comment (bigludo7) [reply code/API_definitions/Call_Forwarding_Signal.yaml:1]: Yes ! done !

File: CHANGELOG.md (hunk 1 above)
Comment (bigludo7) [reply CHANGELOG.md]: Fixed .... but we do not have an issue fir r1.2-rc because not sure where the links points to.

**Comment by FabrizioMoggio** [comment]
> @tanjadegroot For context: @FabrizioMoggio has issue with easyCLA so I copied his PR. I fixed your review comments. Regarding your 2 questions from the yaml:
> 
> _on line 229: does unconditional forwarding overwrite conditional forwarding if both are active ?_ Ludo: I tend to think that unconditional has predominance and override all other but happy to listen from @FabrizioMoggio
> 
> _on line 230: with "This endpoint exceeds the main scope of the CFS API, for this reason an error code 501 can be returned" do you mean that this second endpoint is optional and may not be provided and that 501 is returned in that case ?_ Ludo: Yes

_on line 229: my sentence was actually intended to just say that more than one kind of call forwarding service can be active at the same time and that the API will provide back every kind of service that is configured. The main kind are unconditional and conditional than we can have different flavors of conditional. The logic of what can be configured in the Network and the priorities of the call fwd services have no effect on the API. Anyway I agree,  if the they are both active in the same timeslot, unconditional  takes the precedence. 

-on line 230: some additional information here: https://github.com/camaraproject/CallForwardingSignal/issues/37#issuecomment-2210346948

File: README.md [hunk 3]
```diff
@@ -15,23 +15,22 @@ Repository to describe, develop, document and test the Call Forwarding Signal AP
...
 * Describe, develop, document and test the APIs (with 1-2 Telcos)  
-* Started: February 2024
-* Location: virtually  
+* Started: February 2024 
 
 ## Meetings
 * Meetings are held virtually
```
Comment (FabrizioMoggio) [inline README.md:21]: @bigludo7 we should remove or line 21 or line 23

**Review by FabrizioMoggio — CHANGES_REQUESTED** [review]
README.MD to be updated (we should remove or line 21 or line 23)

File: README.md (hunk 3 above)
Comment (bigludo7) [reply README.md:21]: done

**Review by FabrizioMoggio — APPROVED** [review]
LGTM

**Pull request merged**